etc/api_cfm.yaml

```diff
@@ -0,0 +1,379 @@
+          description: The fridge object in JSON
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  fridge:
+                    $ref: '#/components/schemas/Fridge'
+                  update:
+                    $ref: '#/components/schemas/Update'
+        '404':
+          description: The server can not find the requested resource.
+        '500':
+          description: The server encountered an unexpected condition that prevented it from fulfilling the request.
+        '511':
+          description: Indicates that the client needs to authenticate to gain network access.
+
+  /v1/fridges/{id}/updates:
+    parameters:
+      - in: path
+        name: id
+        schema:
+          type: string
+        required: true
+        description: Database key of the fridge.
+        example: gowanusfridge
+
+    get:
+      summary: Return a list of Update objects for the specified fridge.
+      tags:
+        - update
+      responses:
+        '200':
+          description: A JSON array of Update objects
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Update'
+        '404':
+          description: The server can not find the requested resource.
+        '500':
+          description: The server encountered an unexpected condition that prevented it from fulfilling the request.
+        '511':
+          description: Indicates that the client needs to authenticate to gain network access.
+
+    post:
+      summary: Create a new Update object.
+      tags:
+        - update
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Update'
+      responses:
+        '201':
+          description: Update object was created successfully.
+        '400':
+          description: The request could not be understood by the server due to incorrect syntax.
+        '401':
+          description: Indicates that the request requires user authentication information.
+        '404':
+          description: The Fridge object referred to by the Update object does not exist.
+        '500':
+          description: The server encountered an unexpected condition that prevented it from fulfilling the request.
+
+
+components:
+
+  schemas:
+
+    Update:
+      type: object
+
+      required:
+        - timestamp
+        - operation
+
+      properties:
+        timestamp:
+          description: The ISO formatted date/time of this status update.
+          type: string
+          format: date-time
+          readOnly: true
+          example: "2022-03-29T18:10:38.547Z"
+
+        operation:
+          description: The operational status of the fridge.
+          type: string
+          enum: [working, out of order, not at location]
```

Comment:
Missed this during my review. 

The checkbox should have these options:
`needs cleaning, needs servicing, not at location`

If none of the checkboxes are filled the client could send `working`?

How does that sound?

So the enum could be:

`[working, needs cleaning, needs servicing, not at location]`